src/deploy-cromwell-on-azure/scripts/mount_containers.sh

```diff
@@ -139,7 +139,9 @@ done
 docker_compose_overrides="version: \"3.6\"\nservices:\n  cromwell:\n    volumes:\n$cromwell_volumes  tes:\n    environment:\n      - ExternalStorageContainers=$tes_ext_storage_containers\n"
 echo -e "$docker_compose_overrides" > "docker-compose.override.yml"
 
-# Execute fstab
+# Unmount existing mounts and remount using the updated /etc/fstab
+sudo umount -a -f -t fuse
```

Comment:
Unmounting all of the mounts that were mounted by the system using previously generated /etc/fstab, since it is possible that some containers were removed from containers-to-mount. Clearing the mount.blobfuse.log, so any warnings in it will be actionable, as opposed to keeping the old warnings that might have resolved in the meantime.